test/content-to-slotted-usages/before-fixes.html

```diff
@@ -0,0 +1,18 @@
+<style>
+  x-tabs ::content>x-panel {
+    color: #00F;
+  }
+</style>
+<template>
+  <style>
+    :host ::content > [top-item] {
+      color: #F00;
+    }
+    ::content > top-item {
+      color: #0F0;
+    }
+    ::content > * {
+      color: #00F;
+    }
```

Comment:
I'd like to see a couple examples of `::content` usages here that can't be fixed.

Reply:
The test harness for comparing before and after expects no "incompatible edits"

`assert.deepEqual(incompatibleEdits, []);` Should I modify this to allow for "incompatible edits"?

Reply:
An incompatible edit is one where two replacements (in one edit, or across multiple edits) overlap such that both of them can't be applied.

It's ok if there are warnings without fixes.